terraform/cloud-platform-eks/eks.tf

```diff
@@ -0,0 +1,39 @@
+###############
+# EKS Cluster #
+###############
+
+module "eks" {
+  source  = "terraform-aws-modules/eks/aws"
+  version = "v7.0.0"
+
+  cluster_name       = local.cluster_name
+  subnets            = concat(module.cluster_vpc.private_subnets, module.cluster_vpc.public_subnets)
+  vpc_id             = module.cluster_vpc.vpc_id
+  config_output_path = "./files/"
+
+  worker_groups = [
+    {
+      instance_type        = "m4.large"
+      subnets              = module.cluster_vpc.private_subnets
+      asg_max_size         = 5
+      asg_min_size         = 2
+      asg_desired_capacity = 2
+      key_name             = local.cluster_base_domain_name
+    }
+  ]
+
+  map_users = [
+    {
+      userarn  = "arn:aws:iam::754256621582:user/AlejandroGarrido"
+      username = "AlejandroGarrido"
+      groups   = ["system:masters"]
```

Comment:
Is there any way to avoid having named individuals here? Can we have something that iterates through the members of the ministryofjustice/webops team, or a set of IAM users we pull from the AWS cli?

Reply:
I suppose we will try to implement our kuberos (or kubehook, which is not deprecated) so everyone will be able to connect using auth0. Still, we will need a seed user to apply the initial set up, this seed user is the one which will be listed there.

If we are going to choose the path you mention (our IAM accounts instead of auth0) will be good to do exactly what you said. But still we're going to need a seed user though (can be a generic user, it doesn't need to be an individual or member of our team).

I guess we'll realize which path will be when we start to deploy the cluster components,

Reply:
I'll start to deploy all cluster components we have for the kops and see how it goes with the kuberos